tests/e2e/reports/breadcrumbs.wdio-spec.js

```diff
@@ -37,13 +37,13 @@ describe('Reports tab breadcrumbs', () => {
     },
   };
   const offlineUser = userFactory.build({
-    username: 'offlineuser',
+    username: 'offline-user-breadcrumbs',
```

Comment:
Fixes this one:
![Screen Shot 2022-10-06 at 2 14 53 pm](https://user-images.githubusercontent.com/66472237/194245040-0103a147-c858-4b13-8f55-b318c580fbcf.png)